python/ray/dag/dag_node_operation.py

```diff
@@ -269,13 +269,26 @@ def _build_dag_node_operation_graph(
 
     # Add an edge from WRITE of the writer task to READ of the reader task.
     for dag_idx, task in idx_to_task.items():
+        if (
+            isinstance(task.dag_node, ClassMethodNode)
+            and task.dag_node.is_class_method_output
+        ):
+            # TODO(wxdeng): Handle the case where the task is a class method output.
```

Comment:
which case you plan to handle here?

Reply:
I just `continue` for all `dag_node.is_class_method_output == True` cases, since this part of code does not consider `dag_node.is_class_method_output == True` yet. We might or might not need to handle this case.